### PR TITLE
bgpq4 0.0.7 (new formula)

### PR DESCRIPTION
**Pull request by funzoneq**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Adding support for bgpq4. A fork of bgpq3 that is being actively maintained/developed.

### CODE REVIEW

File: Formula/bgpq4.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  depends_on "automake"
+
+  def install
+    system "./bootstrap"
+    system "./configure", "--prefix=#{prefix}", "--mandir=#{man}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/bgpq4", "AS-ANY"
```
Comment (SMillerDev) [inline Formula/bgpq4.rb:24]: Please check for some output here.

File: Formula/bgpq4.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "autoconf"
+  depends_on "automake"
+
+  def install
+    system "./bootstrap"
```
Comment (SMillerDev) [inline Formula/bgpq4.rb:18]: Does this download anything?

Comment (funzoneq) [reply Formula/bgpq4.rb:18]: No, it just runs autoconf/automake: https://github.com/bgp/bgpq4/blob/main/bootstrap

File: Formula/bgpq4.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    system "./configure", "--prefix=#{prefix}", "--mandir=#{man}"
+    system "make", "install"
+  end
+
+  test do
+    output  = "no ip prefix-list NN\n"
+    output += "! generated prefix-list NN is empty\n"
+    output += "ip prefix-list NN deny 0.0.0.0/0\n"
```
Comment (carlocab) [inline Formula/bgpq4.rb:26]:
```suggestion
    output = <<~EOS
      no ip prefix-list NN
      ! generated prefix-list NN is empty
      ip prefix-list NN deny 0.0.0.0/0
    EOS
```

Comment (funzoneq) [reply Formula/bgpq4.rb]: Updated

File: Formula/bgpq4.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+class Bgpq4 < Formula
+  desc "BGP filtering automation for Cisco, Juniper, BIRD and OpenBGPD routers"
+  homepage "https://github.com/bgp/bgpq4"
+  url "https://github.com/bgp/bgpq4/archive/refs/tags/0.0.7.tar.gz"
+  sha256 "c39af7d2a8b0f4cf61543e776677baaf067aa087181851572211d168258f7f9f"
+  license "BSD-2-Clause"
+  head "https://github.com/bgp/bgpq4.git"
```
Comment (nandahkrishna) [inline Formula/bgpq4.rb:7]:
```suggestion
  head "https://github.com/bgp/bgpq4.git", branch: "main"
```

File: Formula/bgpq4.rb [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "autoconf"
+  depends_on "automake"
```
Comment (nandahkrishna) [inline Formula/bgpq4.rb:15]:
```suggestion
  depends_on "autoconf" => :build
  depends_on "automake" => :build
```

**Review by nandahkrishna — COMMENTED** [review]
A couple of suggestions. Could you squash all your commits into one commit? (Using the same commit message.)

**Comment by funzoneq** [comment]
Suggestions have been processed and the commit has been squashed.

**Review by dawidd6 — APPROVED** [review]

**Comment by nandahkrishna** [comment]
Thanks, @funzoneq!

**Pull request closed without merging**